### PR TITLE
Feature #3

### PR DESCRIPTION
**Pull request by denis**
The user can see and select at least three shapes from the palette, as follows: a square, a circle, a triangle.

Build: http://barushev.net/shaper/builds/3/

What's done:
- Added models.
- Learned about how to draw a square, circle and triangle in SVG.

### CODE REVIEW

File: src/components/Circle.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import * as models from "../models/shapes/circle.ts";
+
+export interface CircleProps {
+  shape: models.Circle;
+}
+
+export class Circle extends React.Component<CircleProps, {}> {
+  render() {
+    const shape = this.props.shape;
```
Comment (danschultz) [inline src/components/Circle.tsx:11]: Tip: you can use destructuring here.

``` ts
const {shape} = this.props;
```

File: src/components/PaletteItem.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    return <div className="palette-item">
+      <svg width={this.props.size} height={this.props.size}>
+        {this.renderShape(this.props.type)}
+      </svg>
+    </div>;
+  }
+
+  private renderShape(type: string) {
+    let size = Number(this.props.size);
+    let center_x = size / 2;
```
Comment (danschultz) [inline src/components/PaletteItem.tsx:25]: Nit: use camel case

**Comment by danschultz** [comment]
👍  LGTM

File: src/components/Circle.tsx (hunk 1 above)
Comment (denis) [inline src/components/Circle.tsx:11]: Thanks for the tip, will use it in later PRs.

File: src/components/PaletteItem.tsx (hunk 2 above)
Comment (denis) [inline src/components/PaletteItem.tsx:25]: You are right, I will refactor later.

**Pull request merged**